Files/ViewModels/SettingsViewModel.cs

```diff
@@ -105,8 +105,12 @@ public static async void OpenLogLocation()
             await Launcher.LaunchFolderAsync(ApplicationData.Current.LocalFolder);
         }
 
-        public static async void OpenThemesFolder() => await NavigationHelpers.OpenPathInNewTab(App.ExternalResourcesHelper.ThemeFolder.Path);
-
+        public static async void OpenThemesFolder()
+        {
+            await Task.Delay(500);
```

Comment:
I'm not so sure this behavior is better than the alternative.
@gave92 what do you think?

Reply:
@yaichenbaum  was the only way I found for him to close the dialog completely and then go to the themes folder.

Reply:
Can you try using `await CoreApplication.MainView.Dispatcher.YieldAsync()` instead?

Reply:
![image](https://user-images.githubusercontent.com/78214358/132882993-845a4061-6a63-4814-8228-921052015261.png)
@hez2010 With the delay, he solved this...

Reply:
> @yaichenbaum was the only way I found for him to close the dialog completely and then go to the themes folder.
> And the best: it works :)

We have some other changes planned that will resolve this issue differently, but it the meantime, we don't want to trade one problem for another 😂

Reply:
So, but does it mean that we are going to use my solution or will it be replaced by your solution?👀

Reply:
I think we should try your solution in the meantime, but we should try to do it without the delay.

Reply:
I will look for alternatives to delay

Reply:
@yaichenbaum I've been looking for alternatives but I can't find another way to do as the Delay currently does....

Reply:
@esibruti can you try the solution @hez2010 suggested?

Reply:
https://user-images.githubusercontent.com/78214358/133153576-fd5b6fd8-5391-4e31-a1c5-e0aebe4e1314.mp4
Method 1 - with delay

https://user-images.githubusercontent.com/78214358/133153573-1428cfee-1254-42be-b514-67eac852a6b2.mp4
Method 2 - @hez2010 sugestion

@yaichenbaum So far I have not been able to find alternatives. Other suggestions?

Reply:
Neither of those actually look too bad.

Reply:
I'll prefer using `Dispatcher.YieldAsync()` than `Task.Delay`(). They both achieve a same thing: yield the UI to respond before doing navigation. `Task.Delay` is too tricky IMO.

Reply:
@yaichenbaum So which option do we use? Or are we going to look for more alternatives?

Reply:
Lets go with `Dispatcher.YieldAsync()`.

Reply:
[https://github.com/files-community/Files/pull/6096/commits/a3dce330c83a01eadf26cf6802f766f798d795d1](url)

Added Dispatcher.YieldAsync()

Reply:
Problem Resolved